public/locales/en.json

```diff
@@ -135,8 +135,9 @@
   "poolShare": "Pool share",
   "showMore": "Show more",
   "pair": "Pair",
-  "capmings": "Campings",
+  "campaings": "Campaings",
```

Comment:
Am I reading this right? campings or campaigns?

Reply:
Fixed, thanks!